providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/resolver/process/InProcessResolver.java

```diff
@@ -79,20 +80,15 @@ public void init() throws Exception {
                 while (true) {
                     final StorageStateChange storageStateChange =
                             flagStore.getStateQueue().take();
-                    switch (storageStateChange.getStorageState()) {
-                        case OK:
-                            onConnectionEvent.accept(new ConnectionEvent(
-                                    ConnectionState.CONNECTED,
-                                    storageStateChange.getChangedFlagsKeys(),
-                                    storageStateChange.getSyncMetadata()));
-                            break;
-                        case ERROR:
```

Comment:
Connectivity events are now managed by the common GrpcConnector component, making it unnecessary to handle them at this point.